embedded_gems/pathogen/pathogen_view_components.gemspec

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pathogen/view_components/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'pathogen_view_components'
+  spec.authors       = ['Bioinformatics and Computational Biology Unit, Public Health Agency of Canada']
```

Comment:
**nitpick (blocking):** Just say `Bioinformatics`

Reply:
Removed in 19d951a